deploy.bat

```diff
@@ -17,4 +17,10 @@ dotnet build Snowflake.Data\Snowflake.Data.csproj -c Release --force -v n /p:Sig
 
 dotnet pack Snowflake.Data\Snowflake.Data.csproj -c Release --force -v n --no-build  --output %ROOT_DIR%
 
+aws s3 cp s3://sfc-eng-jenkins/repository/net/sign-artifact.exe .
+@For /F Delims^= %%G In ('""certutil.exe" -HashFile "sign-artifact.exe" SHA512|"find.exe" /V ":""')Do @Set "SHA=%%G"
+if not %SHA%==94f0b4a78979ded42f7f8c8ce2534691f9f874888bcf7963876f5be881cf6d0ce00e6f8d3e656492249fcfcb890ad656745f2cf68f98e828eb02ded6189a87d4 exit 1

```

Comment:
Maybe we could add some ECHO here to know, what the problem is during deploying?

Reply:
Added a echo when failing due the sha verification